### ISSUE TITLE
convertArrayToJson is missing on Android

### ISSUE THREAD

**Issue by alimek** [SudoPlz/react-native-amplitude-analytics#39]
```
node_modules/react-native-amplitude-analytics/android/src/main/java/com/sudoplz/reactnativeamplitudeanalytics/RNAmplitudeSDK.java:168: error: cannot find symbol
            array.put(convertArrayToJson(readableArray.getArray(i)));
                      ^
  symbol:   method convertArrayToJson(ReadableArray)
  location: class RNAmplitudeSDK
```

i have error like that when trying compile android app with latest version.

After added methods from this gist (https://gist.github.com/viperwarp/2beb6bbefcc268dee7ad) all works. Could your please check this on our end ?

RN 0.57.1

**Comment by Titozzz** [comment]
Yeah 0.2.1 is broken cause of that, I'll revert back to 0.2.0 meanwhile.

**Comment by SudoPlz** [comment]
@Titozzz thanks for bringing that up, I'll try to take a look and revert it or fix the issue.

**Comment by ilydnic** [comment]
Sorry about that, I renamed the function before I created the PR to match the repo better but must have missed one. Please check out: https://github.com/SudoPlz/react-native-amplitude-analytics/pull/40.

**Comment by SudoPlz** [comment]
Should be fixed in 0.2.2, thanks @ilydnic
